sdk/mixedreality/mixed-reality-authentication/test/utils/recordedClient.ts

```diff
@@ -1,12 +1,11 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import "./env";
-import type { RecorderStartOptions } from "@azure-tools/test-recorder";
+import "./env.js";
```

Comment:
please delete the `./env.js` and `env-browser.mts` in favor of the `env` from the test-recorder instead.

Reply:
Removed